docs/eventing/getting-started.md

```diff
@@ -0,0 +1,416 @@
+---
+title: "Getting Started with Eventing"
+#linkTitle: "OPTIONAL_ALTERNATE_NAV_TITLE"
+weight: 9
+type: "docs"
+---
+
+This getting started guide will walk you through all the steps involved in a typical Knative workflow. You will learn how to create, send, and verify Knative events.
+
+## Before you begin
+
+You need:
+
+- A Kubernetes cluster with [Knative Eventing installed](https://knative.dev/docs/install/index.html). 
+
+## Set up components
+
+Before you start to send Knative events, you need to create the components needed to move the events. In this guide, you will be creating components individually, but you can also create components by deploying a [single YAML file](https://raw.githubusercontent.com/akashrv/docs/qs/docs/eventing/samples/hello-world/quick-start.yaml).
+
+### Configure namespace for eventing
+
+This guide uses the default `namespace`, so there is no need to create a new one.
+
+1. Set up the `namespace` for Knative Eventing. To set up the `namespace`, add a label to your `namespace` with this command:
+
+```sh
+kubectl label namespace default knative-eventing-injection=enabled
+```
+
+This label triggers Knative to add `Service Accounts`, `Role Bindings`, and a `Broker` to your `namespace`. You'll learn more about `Brokers` in the next section.
+
+### Examine broker for issues
```

Comment:
Again, we are moving away from what the "users goal/task" is here. The user might need to actual perform the step of "examining the broker" but the real user task is "validating that the broker is running" or "ensuring that the broker is created". 

Its a slight difference but on top of that, mentioning "issues" this early (ie. getting started guide is focused at first time users) might cause concern